Tasks/AzureResourceManagerTemplateDeploymentV3/operations/Subscription.ts

```diff
@@ -0,0 +1,16 @@
+import deployAzureRG = require("../models/DeployAzureRG");
+import armResource = require("azure-arm-rest-v2/azure-arm-subscription");
+import { DeploymentScopeBase } from "./DeploymentScopeBase";
+import { DeploymentParameters } from "./DeploymentParameters";
+
+export class Subscription extends DeploymentScopeBase {
```

Comment:
same here. this does not seem to be required

Reply:
Done